tests/unit/test_models/test_submodels/test_interface/test_li_plating/test_irreversible_plating.py

```diff
@@ -0,0 +1,43 @@
+#
+# Test reversible Li plating submodel
+#
+
+import pybamm
+import tests
+import unittest
+
+
+class TestReversiblePlating(unittest.TestCase):
```

Comment:
I think you have a typo here and you want to change all the `ReversiblePlating` by `IrreversiblePlating`. This would explain why the coverage of irreversible plating is so poor even with these tests.

Reply:
This still needs to be done (minor fix)